doc/sphinx/source/contributing/python-tools.rst

```diff
@@ -0,0 +1,201 @@
+.. _pytools:
+
+Tools for developing with the Python programming language
+=========================================================
+
+This page summarizes auxiliary Python tools that we commonly use to
+develop the project. Note that this page is meant to be a quick index.
+Consult the documentation on each specific tool for details.
+
+Python editors
+--------------
+
+-  The `Spyder editor <https://www.spyder-ide.org/>`__ is good for
+   getting started with scientific Python coding, because of various
+   inspection and interactive features.
+-  `vscode <https://code.visualstudio.com/>`__ is a more full
+   featured editor.
+-  In the long run, the most efficient approach is to learn a terminal
+   based editor such as `vim <https://www.vim.org/>`__. Note that
+   ``vim`` editing modes can be added as extensions to graphical editors
+   such as ``vscode``.
+
+Interactive development
+-----------------------
+
+Python code can be evaluated interactively, which can speed up the
+development.
+
+-  `IPython shell <https://ipython.org/>`__: It is notably nicer to use
+   than the standard interactive interpreter.
+-  `Jupyter notebook <https://jupyter.org/>`__: Interactive development
+   environment running on the browser. Useful for bigger experiments.
+
+
+.. note::
+    When developing :ref:`validphys <vp-index>` related code interactively, be
+    sure to read about the :ref:`API functionality <vpapi>`.
+
+Testing
+-------
+
+
+-  `pytest <https://docs.pytest.org/en/latest/>`__: It is a
```

Comment:
Not sure if intentional but before these were `pytest` in addition of the link (which makes less obvious that it is a link, so not sure if a bug or feature)

Reply:
Apparently you can't have code blocks inside links with rst.

Reply:
Just evaluate markdown inside the rst to get the link :P